rules/scenarios-of-building-the-system/rule.md

```diff
@@ -0,0 +1,93 @@
+---
+type: rule
+title: Scenarios of building the system
+uri: scenarios-of-building-the-system
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan/
+created: 2014-03-14T06:02:00.000Z
+guid: 3021ff26-7bfa-47b5-811d-82bf417fe880
+---
+There are various scenarios for building the system
```

Comment:
@bradystroud @fenix2222 @tiagov8 

Surely this is an archived rule right? 

Also what is 

> the system

😂

Reply:
updated by Igor